db/R/get.trait.data.R

```diff
@@ -82,8 +82,13 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon,
   trait.data.check <- query.traits(spstr, traits, con = dbcon, update.check.only=TRUE)
   traits <- names(trait.data.check)
 
+  # Set forceupdate FALSE if it's a string (backwards compatible with 'AUTO' flag used in the past)
+  if(!is.logical(forceupdate)) {
+    forceupdate <- FALSE
+  }
+  
   # check to see if we need to update
-  if (forceupdate || !as.logical(forceupdate)) {
+  if (!forceupdate) {
```

Comment:
Well, the standard workflow now uses the wrapper function `runModule.get.trait.data`, which sets forceupdate to FALSE if `is.null(settings$meta.analysis$update)` (which is the case if the meta.analysis block, or the update option within it, is missing). 

So in normal use cases, we should expect the argument to be either TRUE or FALSE (and if it's 'AUTO' the preceding expression will have set it to FALSE). Then we only enter this block if `forceupdate==TRUE`. Within this block, if all the files are found, then we'll return early and the rest of the function doesn't get run.